backend/src/lib/routes.ts

```diff
@@ -298,6 +310,109 @@ export class Routes {
       .catch(err => {
         next({ status: 500, errors: err })
       })
+
+    uploadMetadata: RequestHandler = async (req: Request, res: Response, next) => {
+      const id = req.params.hash
+      const body = req.body
+      if (!('filename' in body) || !body.filename) {
+        next({ status: 422, errors: ['Request is missing filename']})
+        return
+      }
+      if (!('hashSum' in body) || body.hashSum.length != 64) {
```

Comment:
Clean code -ohjeissa mainittiin, että ehtoon ei pitäisi laittaa negaatioita. Omasta mielestä koodi toimii hyvin näin, en oikein tajua, miksei pitäisi käyttää negaatioita. Osoitan tällä lähinnä, että kävin vilkaisee niitä juttuja läpi

Reply:
Tarkoitatko tätä? https://github.com/ryanmcdermott/clean-code-javascript#avoid-negative-conditionals
Luulen, että tuossa puhutaan tuon funktion nimeämisestä.